engine/src/fungible_token.rs

```diff
@@ -11,8 +11,10 @@ use crate::prelude::{
 };
 use aurora_engine_sdk::io::{StorageIntermediate, IO};
 
+/// Gas for `resolve_transfer`: 5 TGas
 const GAS_FOR_RESOLVE_TRANSFER: NearGas = NearGas::new(5_000_000_000_000);
-const GAS_FOR_FT_ON_TRANSFER: NearGas = NearGas::new(10_000_000_000_000);
+/// Gas for `ft_on_transfer` transfer: 25 TGas + GAS_FOR_RESOLVE_TRANSFER
+const GAS_FOR_FT_ON_TRANSFER: NearGas = NearGas::new(30_000_000_000_000);
```

Comment:
I don't think this is what we want.

If I understood correctly, it's not that we want to just increase the amount we allocate for this call. Rather, we want to send all the leftover gas to this call, only reserving enough for our own resolve callback afterwards.

Reply:
In any case, for the deposit, we have to manually increase the gas.